torch/export/_trace.py

```diff
@@ -1380,6 +1500,7 @@ def _export(
     _allow_complex_guards_as_runtime_asserts: bool = False,
     _disable_forced_specializations: Optional[bool] = False,
     _is_torch_jit_trace: bool = False,
+    _preserve_ops: Optional[List[OpOverload]] = None,
```

Comment:
just curious, but did we finalize on "separate decomp table + preserve_ops list" being the final API we want for export?

Reply:
Yep i think so